### DISCUSSION TITLE
How to enable full screen

### DISCUSSION THREAD

**Discussion by jindal1979** [unoplatform/uno#11633] [Q&A] [answered]
I want to enable full-screen for my application on Android and iOS. I see there is an internal API:

Microsoft.UI.Xaml.Window.Current.DisplayFullscreen

How can I use this internal API in my application? Is there any alternate way to enable full-screen? Please advise.

**Comment by FrozDark** [comment, marked as answer]
Try this
```
#if __ANDROID__ || __IOS__
            ApplicationView.GetForCurrentView().TryEnterFullScreenMode();
#endif
```
